includes/fbproductfeed.php

```diff
@@ -446,29 +446,7 @@ public function sync_all_products_using_feed() {
 		 * @return int[]
 		 */
 		private function get_product_ids() {
-
-			$post_ids = $this->get_product_wpid();
```

Comment:
It's worth deprecating this `\WC_Facebook_Product_Feed::get_product_wpid` method if we're no longer using it.